theme-tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": false,
-    "module": "commonjs",
+    "module": "node16",
     "moduleResolution": "node16",
```

Comment:
Probably not now, but we should look into updating these and other references I may have missed to not use node16 now that it's long deprecated.

Reply:
I just updated this because TS was mad that `module` and `moduleResolution` didn't match, but it's working for now! We can put more effort into doing whatever the correct thing is at a later date.